src/WCSimEventAction.cc

```diff
@@ -373,13 +373,15 @@ void WCSimEventAction::EndOfEventAction(const G4Event* evt)
       for (int pe = 0; pe < nPoisson; pe++) {
 	G4float time = G4RandGauss::shoot(0.0,10.);
 	G4ThreeVector dir(0, 0, 0);
+  ProcessType_t photcreatorproc = WCSimEnumerations::ProcessTypeStringToEnum("dummy");
```

Comment:
why not just `ProcessType_t photcreatorproc = kUnknownProcess`?

Reply:
👍